src/Entity/DoctrineStorageRatio.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tbbc\MoneyBundle\Entity;
+
+/**
+ * Class DoctrineStorageRatio.
+ */
+class DoctrineStorageRatio
+{
+    private mixed $id = null;
```

Comment:
Not sure if this needs to be mixed. Maybe ?int instead?

Reply:
Same as RatioHistory